src/js/utils/strings.ts

```diff
@@ -36,7 +36,7 @@ export function extension(path: string): string {
         return azureFormatMatches[1];
     }
 
-    path = path.split('?')[0].split('#')[0];
+    path = encodeURI(path).split('?')[0].split('#')[0].split(';')[0];
```

Comment:
Why are you calling `encodeURI` on the input path?

Reply:
I'm not sure this will work if it is possible to have `;` in the file path. You can't have `?` or `?` in a file path, so split works for those.

Reply:
1. To deal with URLs that have non encoded URLs as attribute values (see the test requirements on ticket AC)
2. I can add a defensive check to only run this on paths that start with "http"